test/schemas/form/Person.ChangeDOB.schema

```diff
@@ -0,0 +1,27 @@
+Form({
+  Args: {
+    OldDOB: { field: 'DOB' },
+    NewDOB: { domain: 'DateTime' },
+    LastName: { field: 'FullName.LastName' },
+  },
+  Markup: {
+    Tabs: {
+      control: 'tabs',
+      children: {
+        tab: {
+          control: 'tab',
+          children: {
+            group: {
+              control: 'group',
+              children: {
+                OldDOB: { control: 'date' },
+                NewDOB: { control: 'date' },
```

Comment:
Why do we have OldDOB? If this control will be just for info, we need a way to make it disabled.

Reply:
Same as https://github.com/metarhia/metaschema/pull/178/files#r241332710.